src/ngx_http_lua_socket_tcp.c

```diff
@@ -474,6 +474,7 @@ ngx_http_lua_socket_tcp_connect(lua_State *L)
         switch (lua_type(L, -1)) {
         case LUA_TNUMBER:
             lua_tostring(L, -1);
+            /*-fallthrough*/
```

Comment:
Can we add a space after `/*` and also before `*/`? It would look better than the current form. BTW, have you actually tested this with a gcc 7 on your side?

Reply:
GCC 7 seems to be not smart enough to recognize `-fallthrough` if I put a space before it...
I have tested it with `gcc 7.2`.

Reply:
@spacewander Thanks for trying it out. It seems that we can use the following form instead?

```
/* FALLTHRU */
```

I saw it from the gcc doc below you referenced in #1171:

https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wimplicit-fallthrough